mitosheet/src/utils/formatColumns.tsx

```diff
@@ -1,346 +0,0 @@
-import React from "react"
-import MitoAPI from "../jupyter/api"
-import DropdownItem from "../components/elements/DropdownItem"
-import { getSelectedNumberSeriesColumnIDs } from "../components/endo/selectionUtils"
-import { ColumnID, FormatType, FormatTypeObj, GridState, MitoSelection, SheetData } from "../types"
-import DropdownCheckmark from '../components/icons/DropdownCheckmark'
-import { isNumberDtype } from "./dtypes"
-
-export const FORMAT_DISABLED_MESSAGE = 'You must have at least one Number column selected to adjust the formatting.'
-
-
-const formatCellDataAsStringWithCommas = (cellData: string | number | boolean, decimalPlaces?: number): string => {
-    return Number(cellData).toLocaleString("en-US", {minimumFractionDigits: decimalPlaces, maximumFractionDigits: decimalPlaces})
-}
-
-/*
-    For a cell to be formatted as a number, the cell must only contain valid number symbols
-    and be either a int or float column. 
-
-    Note: The cellData type of a float is a string.
-*/
-export const displayCellAsNumber = (cellData: boolean | string | number, columnDtype: string): boolean => {
```

Comment:
The reason this function was required was because _floats_ were cast to strings (e.g. "2.0") on the backend, so that they would be displayed with the correct number of decimals. 

Notably, this was a decision that was very very easy to change (I literally had to delete 2 lines of code in this PR, in utils.py). The reason the decision was originally made was because we had no formatting infrastructure on the frontend - but since you were adding it, this would have been a good time to remove it.

Once I remove this random cast to a string, we get to remove _all_ this 'displayCellAsNumber' code. This has other benefits later in the `format.tsx` file now (making a lot of other things way less complex and error prone. 

If you're touching formatting, think about what you're formatting, why it is the way that it is, and what you can change about the data you're consuming that will make your life easier!